index.js

```diff
@@ -16,42 +26,36 @@ function stdoutAppender(layout, levels, config) {
     };
     // This is the appender function itself
     return async (loggingEvent) => {
-        const params = JSON.parse(JSON.stringify(botchat_params));
-        if (silentLevel.isLessThanOrEqualTo(loggingEvent.level.levelStr)) {
-            //console.log(`===== silentAlertLevel is less than loggingEvent level.`);
-            if (audioLevel.isLessThanOrEqualTo(loggingEvent.level.levelStr)) {
-                //console.log(`===== log telegram with sound`);
-                Object.assign(params.searchParams, {
-                    text: layout(loggingEvent),
-                    disable_notification: false
-                });
-                try {
-                    await got(sentMessageRoute, params);
-                } catch (error) {
-                    console.error("Error sending to telegram:");
-                    console.error(error);
-                }
-            } else {
-                //console.log(`===== log telegram quietly`);
-                Object.assign(params.searchParams, {
-                    text: layout(loggingEvent),
-                    disable_notification: true
-                });
-                try {
-                    await got(sentMessageRoute, params);
-                } catch (error) {
-                    console.error("Error sending to telegram:");
-                    console.error(error);
-                }
+        if (existStr(config.silentAlertLevel, loggingEvent.level.levelStr) || existStr(config.audioAlertLevel, loggingEvent.level.levelStr)) {
```

Comment:
Can we keep here condition based on log level number, not on exact string? Same goes for detection of disable_notification
It's a more general way in log4js to use log level value and not a strings as far as I know.